Libplanet/Blockchain/BlockChain.cs

```diff
@@ -209,6 +224,23 @@ public Block<T> Tip
             }
         }
 
+        public static Block<T> MakeGenesisBlock(
```

Comment:
Could you write XML comments (docs) for this public method?

Reply:
Okay, I will.